procedures/release-duty/merge-duty/merge_duty.rst

```diff
@@ -64,57 +62,6 @@ first week and then ``m-c`` -> ``m-b`` in the follow-up week.
 Do the prep work a week before the merge
 ----------------------------------------
 
-Set up mergeduty trello tracking board
```

Comment:
We haven't been using the trello board for a few months now and it's gotten much more simplified since it's been automated. Let's remove these docs to avoid confusion.